cypress/e2e/nfts.test.ts

```diff
@@ -46,7 +46,8 @@ describe('Testing nfts', () => {
     cy.get(getTestSelector('nft-details-description-text')).should('exist')
     cy.get(getTestSelector('nft-details-description')).click()
     cy.get(getTestSelector('nft-details-description-text')).should('not.exist')
-    cy.get(getTestSelector('nft-details-toggle-bag')).eq(1).click()
-    cy.get(getTestSelector('nft-bag')).should('exist')
+    // TODO: Enable when universal router is deployed to goerli these tests fail because it is not deployed there
```

Comment:
Can you go into more detail about this failure? Could we instead make the bag or test conditional based off chain instead of removing it entirely?

Reply:
copied what web did and changed the hook to not be enabled on chains where universal router is not deployed an added an enabled flag